src/netaddress.cpp

```diff
@@ -338,9 +339,17 @@ bool CNetAddr::GetIn6Addr(struct in6_addr* pipv6Addr) const
     return true;
 }
 
-// get canonical identifier of an address' group
-// no two connections will be attempted to addresses with the same group
-std::vector<unsigned char> CNetAddr::GetGroup() const
+/**
+ * Get the canonical identifier of our network group
+ *
+ * The groups are assigned in a way where it should be costly for an attacker to
+ * obtain addresses with many different group identifiers, even if it is cheap
+ * to obtain addresses with the same identifier.
+ *
+ * @note No two connections will be attempted to addresses with the same network
+ *       group.
+ */
```

Comment:
this is from some another PR it seems, no?

Reply:
It's from https://github.com/naumenkogs/bitcoin/commit/303372c41a8d5c58a46cf9ed595e30e67bd0bc99 but to prevent merge conflicts, I just added it in